content/language-agnostic/encyclopedia/what-is-an-api.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: "What is an API?"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "APIs" 
+  - "Developer Tools"
+  - "Dependency"
+  - "Interface"
+Catalog Content:  
+  - "https://www.codecademy.com/learn/paths/computer-science"
+  - "https://www.codecademy.com/learn/paths/web-development"
+  - "https://www.codecademy.com/learn/ibm-watson"
+  - "https://www.codecademy.com/learn/learn-alexa"
+---
+
+API stands for Application Programing Interface and is a term used to describe specifications that allow applications to communicate with one another. APIs enable exchange of information, and can be a major source of value( utility, market dependence and revenue) to organizations. APIs are significant components in the evolution of applications because the technical ecosystem is built on top of APIs and leverages them to function and provide many services in use today.
```

Comment:
Let's add a linebreak after the first sentence.

And also type with the open parenthesis `(`.

Reply:
I went ahead and made those changes. Don't hesitate to let me know if you have any more suggestion.

Reply:
LGTM!